docs/runbooks/VirtHandlerRESTErrorsHigh.md

```diff
@@ -23,24 +23,30 @@ Node-related actions, such as starting and migrating workloads, are delayed on t
    $ export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
    ```
 
-2. Check the status of the `virt-handler` pod:
+2. List the available `virt-handler` pods to identify the failing `virt-handler` pod:
 
    ```bash
    $ kubectl get pods -n $NAMESPACE -l=kubevirt.io=virt-handler
    ```
 
-3. Check the `virt-handler` logs for error messages when connecting to the API server:
+3. Check the failing `virt-handler` pod's log for error messages when connecting to the API server:
 
    ```bash
    $ kubectl logs -n  $NAMESPACE <virt-handler>
    ```
 
+   Example error message:
+   
+   ```
+   {"component":"virt-handler","level":"error","msg":"Can't patch node my-node","pos":"heartbeat.go:96","reason":"the server has received too many requests and has asked us to try again later","timestamp":"2023-11-06T11:11:41.099883Z","uid":"132c50c2-8d82-4e49-8857-dc737adcd6cc"}
```

Comment:
Hi, I added "API requests" to the "reason" in the error message just to make it more clear for this PR. The original "reason" message was pulled from k8s apimachinery; if you would like to create an issue for it: https://github.com/kubernetes/apimachinery/blob/fa98d6eaedb4caccd69fc07d90bbb6a1e551f00f/pkg/api/errors/errors.go#L485

Reply:
I believe if that specific error were to get returned by a k8s API call, it will show up in the pod logs for whatever component sent the request so I am assuming they would appear downstream as well but I am not 100% sure.